econplayground/main/models.py

```diff
@@ -321,6 +326,32 @@ class Meta(OrderedModel.Meta):
     a5_min = models.DecimalField(
         max_digits=12, decimal_places=4, default=Decimal('0'))
 
+    a1_2 = models.DecimalField(
+        max_digits=12, decimal_places=4, default=Decimal('0'))
+    a1_max_2 = models.IntegerField(default=10)
```

Comment:
For consistency let's make both the min and max for these values all be the same type: `models.DecimalField(max_digits=12, decimal_places=4)`

So this a1_max_2 field for example, can change to:
```
a1_max_2 = models.DecimalField(max_digits=12, decimal_places=4, default=Decimal('10'))
```

I suppose that in practice, these will pretty much always be a whole number, i.e. an IntegerField. But the value type allows for decimals, and there's no reason we can't accommodate for that here just in case there is a range going from say, 0 to 5.5.

I don't think it's over-engineering too much here, and makes sense to just have all these a-value related fields be stored in the same field type.

Reply:
Actually - sorry, since our RangeEditor doesn't even support setting this to decimal numbers at the moment, maybe both the min and max should be an IntegerField.

Reply:
I think it doesn't matter so much whether these are stored as Decimal or Integer, but I would vote for both the min and max being stored as the same type - unless, did you have a specific reason for having the min be a Decimal and the max be an Integer?

Reply:
The values and the minimums were set to decimal as some of the graphs require that the values do not reach 0. Tom has marked the minimums as 0.1, as such the value can also be set to 0. I think the maximums were also set to DecimalField originally, then it was decided that the maximums at the least would not be decimal values.

Reply:
This is reflected in the properties currently implemented into the model

Reply:
Ah okay, sounds good then. And these types can always be changed in the future I suppose as it's all pretty flexible - I'll merge this as is.